src/components/Modals/Send/utils.ts

```diff
@@ -112,10 +114,13 @@ export const handleSend = async ({
     if (!asset) return ''
     const acccountMetadataFilter = { accountId: sendInput.accountId }
     const accountMetadata = selectPortfolioAccountMetadataByAccountId(state, acccountMetadataFilter)
+    const snapId = getConfig().REACT_APP_SNAP_ID
+    const isSnapsInstalled = await shapeShiftSnapInstalled(snapId)
     // Native and KeepKey hdwallets only support offline signing, not broadcasting signed TXs like e.g Metamask
     if (
       fromChainId(asset.chainId).chainNamespace === CHAIN_NAMESPACE.CosmosSdk &&
-      !wallet.supportsOfflineSigning()
+      !wallet.supportsOfflineSigning() &&
+      !isSnapsInstalled
```

Comment:
this seems like it needs to be like "is wallet metamask AND snap isn't installed"?

Reply:
22621cdfc381c1330aa27da4bbad8334bc80d